src/components/berths/Berth/Berth.scss

```diff
@@ -51,18 +51,16 @@ $berthImageHeight: 12em;
       height: 14px;
       margin-right: 0.3em;
       border-radius: 50%;
+      background-color: $hel-white;
 
-      &.default {
-        background-color: $hel-white;
-      }
-      &.red {
-        background-color: $hel-brick;
+      &--1 {
```

Comment:
I don't like this, you should never tie any styles (or anything in code) to ids. But, I guess it can't be helped this time :/

Reply:
doesn't berth have any other property this style could be tied to?